### ISSUE TITLE
Error when cloning this repo

### ISSUE THREAD

**Issue by mkiol** [MycroftAI/mimic3-voices#12]
Hello, I would just like to report that currently this repository cannot be cloned due to the following error:

```
This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
```

The URLs to onnx files don't work as well.

**Comment by JarbasAl** [comment]
Mycroft is dead and won't be paying any bills....

i think the models need to be hosted somewhere else, if anyone still has access to them?

NeonAI should have access to Mycroft's github and maybe the means to resolve this, ping @NeonDaniel

**Comment by NeonDaniel** [comment]
> NeonAI should have access to Mycroft's github and maybe the means to resolve this, ping @NeonDaniel

Looks like I actually don't have access to this one, but @synesthesiam might. LFS by default has a 1GB monthly quota; I'm not sure when this org's billing date is set, but when that resets the models will be downloadable again from here. I'll see if there's anything we can do on our end.

**Comment by rbrisita** [comment]
Ah, yeah [GitHub link to bandwidth usage](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage). I don't think HuggingFace has bandwidth issues.

**Comment by synesthesiam** [comment]
I've moved on to working on Piper: https://github.com/rhasspy/piper

Its models are all on Huggingface to avoid this exact problem 😄

I don't know if I'm allowed to post the Mimic 3 models there too.

**Comment by rbrisita** [comment]
I had an issue with [Piper](https://github.com/rhasspy/piper/issues/500), and switched to Mimic3 Debian install to resolve. I was [uploading the voice I use](https://huggingface.co/rbrisita/mimic3-voices) to HuggingFace and it seems this person uploaded all if not most of the [voice models](https://huggingface.co/mukowaty/mimic3-voices).

**Comment by mkiol** [comment]
> this person uploaded all if not most of the voice models

Yes, that was me. The app I'm developing [Speech Note](https://github.com/mkiol/dsnote) uses these models, so I had to find a quick solution to make it work again. I have to say that I didn't ask anyone for permission. The license of this repo does not prohibit this, so I thought it was allowed.

**Comment by krisgesling** [comment]
Yes, under the CC BY-SA 4.0 license anyone is free to redistribute these models as is, or in a modified form as long as there's attribution (which you've done) and a description of any changes made. 👍
